src/store/index.js

```diff
@@ -22,4 +24,31 @@ export function getMessageSubject(message: Message): ?string {
   return subjectHeader && subjectHeader.value;
 }
 
+export function getMessageSender(message: Message): ?string {
+  const { headers } = message.payload;
+  const emailSender = headers.find(header => header.name === 'From');
+  return emailSender && emailSender.value;
+}
+
+export function getMessageTimestamp(message: Message): ?string {
+  const { headers } = message.payload;
+  const emailTimestamp = headers.find(header => header.name === 'Date');
```

Comment:
While you won’t have a problem with this sample data, it’s technically not a good idea to trust the Date header since it can be faked. But additionally it’s nice to be able to format the date differently based on locale or time since (i.e. 10:34 AM if it’s today but 10/23/17 if it’s last year) That’s why the API provides `internalDate`